Jenkinsfile_CNP

```diff
@@ -45,7 +45,14 @@ def secrets = [
                 secret('idam-solicitor-username', 'IDAM_SOLICITOR_USERNAME'),
                 secret('idam-solicitor-password', 'IDAM_SOLICITOR_PASSWORD'),
                 secret('idam-systemupdate-username', 'IDAM_SYSTEM_UPDATE_USERNAME'),
-                secret('idam-systemupdate-password', 'IDAM_SYSTEM_UPDATE_PASSWORD')
+                secret('idam-systemupdate-password', 'IDAM_SYSTEM_UPDATE_PASSWORD'),
+                secret('idam-caa-username', 'IDAM_CAA_USERNAME'),
+                secret('idam-caa-password', 'IDAM_CAA_PASSWORD'),
+                secret('idam-mca-client-secret', 'IDAM_MCA_CLIENT_SECRET'),
+                secret('manage-case-s2s-key', 'MANAGE_CASE_S2S_KEY'),
+                secret('idam-noc-approver-username', 'IDAM_NOC_APPROVER_USERNAME'),
+                secret('idam-noc-approver-password', 'IDAM_NOC_APPROVER_PASSWORD'),
+                secret('notify-mca-api-key', 'NOTIFY_MCA_API_KEY')
```

Comment:
have these been added to all environments?

Reply:
from the helm chart changes it looks like these are only needed for manage org? If so they arent needed in the jenkins pipeline

Reply:
They'll be needed for manage case if we ever ended up needing to use it. Not this time but possible in the future. So yeah will need to add them to other envs, will do today

Reply:
Would they be needed in the jenkins pipeline? If so, what for? I would remove them until we need them